src/Acuminator/Acuminator.Vsix/AcuminatorVSPackage.vsct

```diff
@@ -34,6 +37,7 @@
         <Strings>
           <CommandName>cmdidFormatBQLCommand</CommandName>
           <ButtonText>Format BQL Statements</ButtonText>
+		  <LocCanonicalName>.Acuminator.FormatBQL</LocCanonicalName>
```

Comment:
The same with tabs/spaces:
![image](https://user-images.githubusercontent.com/1544021/151838664-3e33bf99-7402-4546-b586-aa8fa57bbd83.png)

Reply:
I'll fix the indentation.